Tasks/SqlAzureDacpacDeploymentV1/task.json

```diff
@@ -16,8 +16,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 225,
-        "Patch": 1
+        "Minor": 231,
```

Comment:
You should bump up the version even in other PowerShell tasks that are referencing VstsAzureRestHelpers_.

Reply:
Yes, I'm trying to do it, but getting some errors for Node 20 generated tasks.

Reply:
Done